### PR TITLE
Support debug and ctop commands in CLI

### PR DESCRIPTION
**Pull request by aylei**
This PR add `debug` and `ctop` commands in the CLI, some examples:

```shell
$ tkc list -A
NAMESPACE            NAME                 PD    TIKV   TIDB   AGE
stability-cluster1   stability-cluster1   3/3   3/3    2/2    26h

$ tkc use -n=stability-cluster1 stability-cluster1
Tidb cluster switched to stability-cluster1/stability-cluster1

$ tkc get tikv
NAME                        READY   STATUS    MEMORY          CPU   RESTARTS   AGE   NODE
stability-cluster1-tikv-0   2/2     Running   2098Mi/4196Mi         0          26h   172.16.4.153
stability-cluster1-tikv-1   2/2     Running   2098Mi/4196Mi         0          26h   172.16.4.151
stability-cluster1-tikv-2   2/2     Running   2098Mi/4196Mi         0          26h   172.16.4.150

# Now, assume that stability-cluster1-tikv-0 is unhealthy, we can use `ctop` to view the real-time metrics
$ tkc ctop stability-cluster1-tikv-0
# Or see the node metrics of tikv-0
$ tkc ctop node/172.16.4.153

# If the problem still exists, we can debug the tikv instance
$ tkc debug stability-cluster1-tikv-0 

████████╗██╗██████╗ ██████╗      ██████╗████████╗██╗
╚══██╔══╝╚═╝██╔══██╗██╔══██╗    ██╔════╝╚══██╔══╝██║
   ██║   ██╗██║  ██║██████╔╝    ██║        ██║   ██║
   ██║   ██║██║  ██║██╔══██╗    ██║        ██║   ██║
   ██║   ██║██████╔╝██████╔╝    ╚██████╗   ██║   ███████╗
   ╚═╝   ╚═╝╚═════╝ ╚═════╝      ╚═════╝   ╚═╝   ╚══════╝

Welcome to tidb cluster control! (https://github.com/pingcap/tidb)

→ / ps -ef
PID   USER     TIME  COMMAND
    1 root      3:52 /tikv-server --pd=stability-cluster1-pd:2379 --advertise-a
  103 root      0:00 bash -l
  112 root      0:00 ps -ef

# we may chroot the tikv container's root filesystem
→ / chroot /proc/1/root

→ / ls
bin            home           mnt            sbin           tmp
dev            lib            proc           srv            usr
entrypoint.sh  lib64          root           sys            var
etc            media          run            tikv-server
```

I will open another PR to add more documents and tests.

Signed-off-by: Aylei <rayingecho@gmail.com>

### CODE REVIEW

File: misc/images/tidb-debug/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    && rm -rf tidb-latest-linux-amd64.tar.gz tidb-latest-linux-amd64
+
+RUN wget https://github.com/brendangregg/FlameGraph/archive/master.zip \
+    && unzip master.zip \
+    && mv FlameGraph-master /opt/FlameGraph \
+    && rm master.zip
+ADD run_flamegraph.sh /run_flamegraph.sh
+
+# used for go pprof
+ENV GOLANG_VERSION 1.10
```
Comment (tennix) [inline misc/images/tidb-debug/Dockerfile]: The Go language needs to be upgraded to 1.12.

File: misc/images/tidb-debug/Dockerfile [hunk 2]
```diff
@@ -37,7 +37,7 @@ RUN wget https://github.com/brendangregg/FlameGraph/archive/master.zip \
 ADD run_flamegraph.sh /run_flamegraph.sh
 
 # used for go pprof
-ENV GOLANG_VERSION 1.10
+ENV GOLANG_VERSION 1.12
 ENV GOLANG_DOWNLOAD_URL https://golang.org/dl/go$GOLANG_VERSION.linux-amd64.tar.gz
 ENV GOLANG_DOWNLOAD_SHA256 b5a64335f1490277b585832d1f6c7f8c6c11206cba5cd3f771dcb87b98ad1a33
```
Comment (tennix) [inline misc/images/tidb-debug/Dockerfile]: The sha256 checksum also needs to be updated.

**Comment by aylei** [comment]
@tennix @weekface @xiaojingchen PTAL again

**Comment by weekface** [comment]
This looks good to me.

**Pull request merged**